classes/class-woothemes-sensei-certificates.php

```diff
@@ -1155,6 +1162,40 @@ public function get_certificate_url( $course_id, $user_id ) {
 
 	} // End get_certificate_url()
 
+	/**
+	 * Get ID for certificate.
+	 *
+	 * @since  4.16.0
```

Comment:
You should use the `$$next-version$$` placeholder instead of the version number:
```
@since $$next-version$$
```

Reply:
TIL! Upated in 2ccaa84